vtex-sites/storeframework.store
# feat: Add CMS to Alert

## What's the purpose of this pull request?

This PR aims to integrate `Alert` with the CMS, by adding its properties.

## How does it work?

In the file `sections.json `we can define all the editable properties for each component. For the `Alert`, the properties are `icon`, `content` and `link` (`text` and `to`). Also some adjustment had to be made in `Alert` component, this was done in `gatsby.store` in [this commit](https://github.com/vtex-sites/gatsby.store/commit/57be96b7bd997685b01638d47699e31cf8753c1f) and cherry-picked to this current PR. 

**This is the final result in the CMS:** 
![alert cms](https://user-images.githubusercontent.com/10616243/167920059-54e4637e-9351-4ee9-9ea4-ca45946b41af.gif)


**This is how the Alert will be rendered in the page:** 
![alert localhost](https://user-images.githubusercontent.com/10616243/167920071-6b237937-9afc-4c67-b764-01aebfe2fd4a.gif)


## How to test it?

- Set your workspace to `israel` with the command `vtex use israel`
- Sync your project with the CMS watch `vtex cms sync --watch`
- Open the CMS to edit the data in the [developer workspace](https://israel--storeframework.myvtex.com/admin/new-cms/faststore/home/edit/ad2fd81d-a53c-4281-8d01-a4fc2f274db3/)
- On `gatsby.config.ts` change the workspace from `master` to `israel` (so your localhost will fetch data from `israel` workspace when you run the project)
- Run your project with `yarn develop` to see the data you've edited in the CMS (P.S.:  Every time you edit things in CMS, you have to rerun the project to get the new data)

## References

Related gatsby.store [PR](https://github.com/vtex-sites/gatsby.store/pull/29)
Related Jira [task](https://vtex-dev.atlassian.net/browse/FSSS-304)
